WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -1926,3 +1943,17 @@ private extension ReaderStreamViewController {
         static let contentErrorImage = "cloud"
     }
 }
+
+extension ReaderStreamViewController: ReaderPostCardCellDelegate {
+    func heightDidChange() {
+        WPAnalytics.track(.readerChipsMoreToggled)
```

Comment:
I think this needs to be in the place where the tap actually happens.

Otherwise, if any other place of the code starts using the chips view, it will have to implement this event.

Also, is quite strange to see this event in height change callback.

Reply:
Done in: 5c0b1a884edba451763d1302719df988678ba4b7